packages/components/pie-toggle-switch/src/index.ts

```diff
@@ -35,28 +38,34 @@ export class PieToggleSwitch extends RtlMixin(LitElement) implements ToggleSwitc
 
     render () {
         const {
+            aria,
             isChecked,
             isDisabled,
         } = this;
 
+        const toggleSwitchId = 'toggle-switch-description';
+
         return html`
             <label
                 data-test-id="toggle-switch-component"
                 class="c-toggleSwitch"
                 ?isChecked="${isChecked}"
-                ?isDisabled=${isDisabled}>
+                ?isDisabled=${isDisabled}
+                aria-label="${aria?.label || null}">
```

Comment:
I think instead of null you can use `nothing` that won't render the aria-label if it's not provided i believe.